app/services/state_file/state_information_service.rb

```diff
@@ -74,6 +74,7 @@ def state_code_to_name_map
         vita_link: "https://airtable.com/appnKuyQXMMCPSvVw/pag0hcyC6juDxamHo/form",
         voucher_form_name: "Form AZ-140V",
         voucher_path: "/pdfs/AZ-140V.pdf",
+        optional: false
```

Comment:
no change necessary, but now that i look at this, i wonder if calling this key `schema_optional` might make it clearer how it's used